opm/input/eclipse/EclipseState/Grid/EclipseGrid.hpp

```diff
@@ -114,6 +114,22 @@ namespace Opm {
             return activeIndex(globalIndex);
         }
 
+        const std::vector<std::string> get_all_lgr_labels() const {
+            std::vector<std::string> sliced_labels(all_lgr_labels.begin() + 1, all_lgr_labels.end());
+            return sliced_labels;
+        }
+
+        const std::vector<std::string> get_all_labels() const {
+            return all_lgr_labels;
+        }
+        
+        const std::string get_lgr_tag() const {
+            return this->lgr_label;
+        }
```

Comment:
Returning `const` **objects** is typically not useful.  This is very different from when you're returning **references** however.  Those very well can, and often should, be `const`.